views/layouts/dashboard.handlebars

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>NODOMON</title>
+  <link href="https://fonts.googleapis.com/css?family=IBM+Plex+Sans:400,400i,700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="/css/style.css">
+</head>
+
+<body class="flex-column min-100-vh">
+  <header class="hero">
+  </header>
+  <nav class="navbar">
+    <ul>
+      <li class="li-inline nav-item"><h1 class="app-title"><a href="/">NODOMON-CHOICE-RPG</a></h1></li>
+      <li class="li-inline nav-item"><a href="/">Home</a></li>
+      <li class="li-inline nav-item"><a href="/Sign up">sign up</a></li>
+      <li class="li-inline nav-item"><a href="/login">Login</a></li>
+      <li class="li-inline nav-item"><a href="/adventure">adventure</a></li>
+      <li class="li-inline nav-item"><a href="/change character">change character</a></li>
```

Comment:
Remove change and capitalize character to Characters

I'm thinkiing we keep this plural